### ISSUE TITLE
YouTube Feeds Not Visible

### ISSUE THREAD

**Issue by compoundsnake** [spocky/miproja1#58]
Hello,

I have both accessibility controls and notifications available, but for whatever reason the YouTube channels for "Recommended" and "Subscriptions" are not visible even when toggled on. I can still make generic channels for YouTube visible (e.g., "Trending") but these used-specific channels do not appear and I'm not sure why. There was a Reddit thread where others noted the same issue.

Apologies for any confusion and thanks!

**Comment by spocky** [comment]
Hi,

You seem to say that other people had the same issue. Could your please share how you fixed it so that it can help others ?
